app/models/la_funded_place.rb

```diff
@@ -1,5 +1,9 @@
 class LaFundedPlace < CompulsorySchool
   def institution_type
-    'funded_places'
+    'local authority'
```

Comment:
```suggestion
    'local_authority'
```

I'm pretty sure this should be underscored.

Reply:
Good spot